flink/content.md

```diff
@@ -0,0 +1,95 @@
+# What is Apache Flink?
+
+Apache Flink is an open source stream processing framework with powerful stream- and batch-processing capabilities.
+
+Learn more about Flink at [https://flink.apache.org/](https://flink.apache.org/)
+
+> [wikipedia.org/wiki/Apache_Flink](https://en.wikipedia.org/wiki/Apache_Flink)
+
+%%LOGO%%
+
+# How to use this Docker image
+
+## Run a Flink local cluster
+
+To run a single Flink local cluster:
+
+```console
+$ docker run --name flink_local -p 8081:8081 -t flink local
+```
+
+Then with a web browser go to `http://localhost:8081/` to see the Flink Web Dashboard (adjust the hostname for your Docker host).
+
+To use Flink, you can submit a job to the cluster using the Web UI or you can also do it from a different Flink container, for example:
+
+```console
+$ docker run -t flink flink run -m <jobmanager:port> -c <your_class> <your_jar> <your_params>
```

Comment:
Might be a good idea to add `--rm` to this one, since it's not long-running. :+1: